fire/cli/ts/hts.py

```diff
@@ -97,3 +97,112 @@ def hts(objekt: str, parametre: str, fil: click.Path, **kwargs) -> None:
     _udtræk_tidsserie(objekt, HøjdeTidsserie, HTS_PARAMETRE, parametre, fil)
 
     return
+
+@ts.command()
+@click.argument("objekt", required=True, type=str)
+@click.option(
+    "--plottype",
+    "-t",
+    required=False,
+    type=click.Choice(["rå", "fit", "konf"]),
+    default="rå",
+    help="Hvilken type plot vil man se?",
+)
+@click.option(
+    "--parametre",
+    "-p",
+    required=False,
+    type=str,
+    default="kote",
+    help="Hvilken parameter skal plottes?",
+)
+@click.option(
+    "--minpunkter",
+    "-N",
+    required=False,
+    type=int,
+    default = 3,
+    help="Minimum antal punkter i tidsserien.",
+)
+@fire.cli.default_options()
+def plot_hts(objekt: str, plottype: str, parametre: str, minpunkter, **kwargs) -> None:
+    """
+    Plot en Højdetidsserie.
+
+    Et simpelt plot der som standard viser kotens udvikling over tid.
+
+    "OBJEKT" kan enten være et Højdetidsserie ID eller navnet på en Punktsamling.
+    Er "OBJEKT" en Punktsamling, plottes alle tidsserier relateret til Punktsamlingen.
```

Comment:
Kan du lave et eksempel, inkl. figur, for det også?

Reply:
Ah det har jeg glemt at slette. Kommandoen plotter ikke længere en punktsamling, men til gengæld kan man gøre det med analyse-hts. Får rettet teksten.